src/cargo/ops/cargo_rustc/job_queue.rs

```diff
@@ -181,7 +181,7 @@ impl<'a> JobQueue<'a> {
         });
 
         // Print out some nice progress information
-        try!(self.note_working_on(config, key.pkg, fresh));
+        try!(self.note_working_on(config, key.pkg, fresh, &key.profile.doc));
```

Comment:
This should actually be fine to just pass the whole `Key` into the function below and then it can decide which part to access